Don't consider venv as externally managed

krisl opened the pull request:
Allow installation into virtual environments without requiring `--break-system-packages`

nuclearsandwich reviewed (commented):
Thanks for the PR. Since you contributed a test for this function in #1000 is it worth rebasing now that it has merged and covering the virtualenv case in the test?

krisl commented:
> Thanks for the PR. Since you contributed a test for this function in #1000 is it worth rebasing now that it has merged and covering the virtualenv case in the test?

cool, I rebased and added a test in there
